src/frontend/package.json

```diff
@@ -59,6 +59,7 @@
     "faker": "5.5.3",
     "fetch-mock": "9.11.0",
     "jest": "26.6.3",
+    "jest-matchmedia-mock": "1.1.0",
```

Comment:
This lib seems not used.

Reply:
As matchmedia is used by hot-toast, and doesn't exists in jsdom, we need to mock it.